example/custom_sequencer/BUILD.bazel

```diff
@@ -0,0 +1,31 @@
+load("@aspect_rules_jest//jest:defs.bzl", "jest_test")
+
+# Case 1: auto_configure_test_sequencer is True so custom test sequencer will be overridden
+jest_test(
```

Comment:
Even though it's defaulted, perhaps it's clearer to explicitly set it to `True` for this test.

Reply:
👍